www/i18n/en.json

```diff
@@ -128,13 +128,13 @@
     "to-label": "To Label",
     "show-all": "All Trips",
     "no-trips-found": "No trips found",
-    "choose-mode": "Mode 📝 ",
-    "choose-replaced-mode": "Replaces 📝",
-    "choose-purpose": "Purpose 📝",
-    "choose-survey": "Add Trip Details 📝 ",
-    "select-mode-scroll": "Mode (👇 for more)",
-    "select-replaced-mode-scroll": "Replaces (👇 for more)",
-    "select-purpose-scroll": "Purpose (👇 for more)",
+    "choose-mode": "Mode",
+    "choose-replaced-mode": "Replaces",
+    "choose-purpose": "Purpose",
+    "choose-survey": "Add Trip Details",
+    "select-mode-scroll": "Mode (scroll for more)",
+    "select-replaced-mode-scroll": "Replaces (scroll for more)",
+    "select-purpose-scroll": "Purpose (scroll for more)",
```

Comment:
We should make these changes for other languages too, right?

Reply:
I created a PR for Spanish and Lao! https://github.com/e-mission/e-mission-translate/pull/53